scripts/update-npm-tag.mjs

```diff
@@ -0,0 +1,53 @@
+import shell from 'shelljs';
+import { getPackageInfos } from "workspace-tools";
+
+function getPublicPackagesNames() {
+  const packages = getPackageInfos(import.meta.resolve('..'));
+  return Object.values(packages)
+    .filter((pkg) => !pkg.private)
+    .map((pkg) => pkg.name);
+}
+
+async function getLatestReleaseCandidate(packageName, distTag) {
+  const response = await fetch(`https://registry.npmjs.org/${packageName}`);
+  const packageDetails = await response.json();
+
+  const filteredVersions = Object.entries(packageDetails.time)
+    .filter(([version]) => version.includes(distTag))
+    .sort(([, dateTimeA], [, dateTimeB]) => new Date(dateTimeB).getTime() - new Date(dateTimeA).getTime());
+
+  if (filteredVersions.length === 0) {
+    throw new Error(`No release candidate found with dist-tag ${distTag}`);
+  }
+
+  return filteredVersions[0][0];
+}
+
+async function run() {
+
+  // Get the NPM dist-tag parameter
+  const distTag = process.argv[2];
+
+  if (!distTag) {
+    throw new Error("Please provide a NPM dist-tag as a parameter");
+  }
+
+  // Resolve the public packages of the repository
+  const packagesNames = getPublicPackagesNames();
+
+  // Fetch the latest release of the received dist-tag
+  const latestVersion = await getLatestReleaseCandidate(packagesNames[0], distTag);
```

Comment:
This is just to know what the latest version of all packages is, right?

Reply:
Yes. The idea is that we have just released a new version this script is not aware of and we need to find about it.
This works because all packages share the same version.

Reply:
@tdeekens Your comment got me thinking and I've discovered that not all repositories use the same version for all their public packages so I've updated the latest version resolution to be per package: e344e40e917d0b3711f0b31897d033088b7e11bd